components/calendar/src/cal/hijri.rs

```diff
@@ -62,6 +62,7 @@ pub struct HijriSimulated {
     data: Option<DataPayload<ErasedMarker<HijriData<'static>>>>,
 }
 
+#[non_exhaustive]
```

Comment:
these are both private and this will conflict with https://github.com/unicode-org/icu4x/pull/6404